pkg/storage/mvcc_history_test.go

```diff
@@ -160,10 +160,12 @@ func TestMVCCHistories(t *testing.T) {
 	const statsTS = 100e9
 
 	datadriven.Walk(t, testutils.TestDataPath(t, "mvcc_histories"), func(t *testing.T, path string) {
+		disableSeparateEngineBlocks := strings.Contains(path, "_disable_separate_engine_blocks")
```

Comment:
Why not shadow `separateEngineBlocks`? As is, it's a bit annoying to have to carry this new bool into every new conditional.

```
separateEngineBlocks := separateEngineBlocks && !strings.Contains(path, "_disable_separate_engine_blocks")
```

Reply:
I thought our linters frowned on that sort of thing?

Reply:
Yup:

```
pkg/storage/mvcc_history_test.go:163:3: declaration of "separateEngineBlocks" shadows declaration at line 59 (shadow)
```